bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -657,7 +665,15 @@ attributes from the config_cache.xml file (with keys converted to upper-case).
 
 <!-- Can you interpolate from an initial condition file at startup (is there a finidat file to find)? -->
 <!-- NOTE: Make sure all possible lnd_tuning_mode options are accounted for. Some finidat files are only for one mode -->
-<use_init_interp  use_cndv=".false." use_fates=".false." sim_year="1850"                                >.true.</use_init_interp>
+<use_init_interp use_cndv=".false." use_fates=".false." sim_year="1850" lnd_tuning_mode="clm4_5_GSWP3v1" >.true.</use_init_interp>
```

Comment:
As in #2427, simplifying this should fix the ERI test issue as well make this easier to read.